flood_adapt/integrator/sfincs_adapter.py

```diff
@@ -21,11 +22,11 @@ def __init__(self, model_root: Optional[str] = None):
         # if validate_existence_root_folder(model_root):
         #    self.model_root = model_root
 
-        self.sf_model = SfincsModel(root=model_root, mode="r+")
+        # self.sf_model = SfincsModel(root=model_root, mode="r+")
```

Comment:
Do we have a fix for the hydromt issue yet?

We only need to comment out the import of the SfincsModel from hydromt. The rest can remain in the script. At least this worked for me

Reply:
we install the correct hydromt version with the environment yml file to make sure that it works.